services/workflows-service/src/common/filters/AllExceptions.filter.ts

```diff
@@ -29,12 +29,14 @@ export class AllExceptionsFilter extends BaseExceptionFilter {
       this.logError(request, serverError);
     }
 
+    const error = serverError.getResponse();
+
     response
       .status(serverError.getStatus())
       .setHeader('Content-Type', 'application/json')
       .json({
+        ...(typeof error === 'string' ? { message: error } : error),
```

Comment:
Did you confirm how this change affects the collection flow and backoffice?

Reply:
will do it on a different PR which take care for validation issues 
https://teamballerine.slack.com/archives/C0510CD1R8T/p1706629193802419